recipes/scikit-video/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "sk-video" %}
+{% set version = "1.1.9" %}
+{% set sha256 = "812779b5245e9eaa330ba04fa2bc050b11f1d17246050da4365386d993108c49" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
+    - scipy
```

Comment:
It doesn't look like `scipy` is required to build the package.

Reply:
looks like it is: I tried to remove it but the build fails

Reply:
Oops - missed the self import in the `setup.py`.